src/Files.App/Helpers/Win32/Win32Helper.Storage.cs

```diff
@@ -11,8 +11,16 @@
 using System.Runtime.InteropServices.ComTypes;
 using System.Text;
 using System.Windows.Forms;
+using Vanara;
 using Vanara.PInvoke;
 using Windows.System;
+using Windows.Win32;
+using Windows.Win32.Foundation;
+using Windows.Win32.Storage.FileSystem;
```

Comment:
If you struggled with co-existence of CsWin32 and Vanara, I don't mind to create a new file `Win32Helper.CsWin32.cs` that only contains CsWin32 references so we can do replacement of Vanara easily. However, as it depends upon you, I'll leave it up to you.